mkosi/__init__.py

```diff
@@ -3756,6 +3756,25 @@ def file_list(root: str) -> Generator[str, None, None]:
                 queue.append(entry.path)
 
 
+def cpio_compressor_command(option: Union[None, str, bool]) -> List[str]:
```

Comment:
shouldn't we share this with the tar codepaths? i.e. if we invoke the compressor manually for the cpio case there shouldn't be any reason why we couldn't do this for the tar case the exact same way?

Reply:
Yeah, it would be nice to share this. I didn't do that so far because that'll require a bigger restructuring of how `tar` is invoked.